.code-samples.meilisearch.yaml

```diff
@@ -467,16 +467,19 @@ getting_started_add_documents_md: |-
   curl \
     -X POST 'http://localhost:7700/indexes/movies/documents?primaryKey=id' \
     -H 'Content-Type: application/json' \
+    -H 'Authorization: Bearer aSampleMasterKey' \
     --data-binary @movies.json
   ```
 getting_started_check_task_status: |-
   curl \
     -X GET 'http://localhost:7700/tasks/0'
+    -H 'Authorization: Bearer aSampleMasterKey'
 getting_started_search_md: |-
   ```bash
   curl \
     -X POST 'http://localhost:7700/indexes/movies/search' \
     -H 'Content-Type: application/json' \
+    -H 'Authorization: Bearer aSampleMasterKey' \
```

Comment:
I suggesting a master key on the `/search` route not a security risk?

Reply:
Good point @bidoubiwa . On the other hand I think requesting API keys might make the quick start too long. @guimachiavelli Maybe we can add a note explaining that in real production situations, the master key should not be used in this way?